LICENSE-3rdparty.csv

```diff
@@ -0,0 +1,2 @@
+Component,Origin,License,Copyright
+datadog-agent,https://github.com/DataDog/datadog-agent,Apache-2.0,"Copyright (c) 2016-present, Datadog, Inc"
```

Comment:
Should this say `datadog-agent-macos-build`?

Reply:
Yes your point makes sense ! Will ask a second opinion before merging without/with it though !

Reply:
Open source team settled the questions, and told me to keep it. Leaving it like that then 🫥